samples/sensor/mcux_acmp/sample.yaml

```diff
@@ -2,7 +2,7 @@ sample:
   description: Demonstration of the NXP MCUX ACMP sensor driver
   name: NXP MCUX ACMP sample
 common:
-  platform_allow: twr_ke18f
+  platform_allow: twr_ke18f mimxrt1170_evk_cm7 mimxrt1170_evk_cm4
```

Comment:
If CM4 is enabled for testing, should there be a board overlay for it as well?

Reply:
> here be a board overlay fo

Yes， both cm4 and cm7 overlay added.